frontend/packages/portal-frontend/src/compound/tiles/HeatmapTile/HeatmapTile.tsx

```diff
@@ -0,0 +1,119 @@
+import { sortHeatmapByViability } from "src/compound/heatmapTab/heatmapPlotUtils";
+import DoseTriangleLabel from "./DoseTriangleLabel";
+import ErrorLoading from "./ErrorLoading";
+import TopLinesMiniTable from "./TopLinesMiniTable";
+
+interface HeatmapTileProps {
+  compoundName: string;
+  isLoadingDataset: boolean;
+}
+
+export const HeatmapTile: React.FC<HeatmapTileProps> = ({
+  compoundName,
+  isLoadingDataset,
+}) => {
+  const {
+    tableFormattedData,
+    doseColumnNames,
+    error,
+    isLoading,
+  } = useDoseTableDataContext();
+
+  const { heatmapFormattedData } = useHeatmapData(
+    tableFormattedData,
+    doseColumnNames
+  );
+
+  // Sort data by ascending mean viability
+  const sortedHeatmapFormattedData = useMemo(
+    () => sortHeatmapByViability(heatmapFormattedData),
+    [heatmapFormattedData]
+  );
+
+  const heatmapTabHref = (() => {
+    const url = new URL(window.location.href);
+    url.searchParams.set("tab", "heatmap");
+    return url.pathname + url.search;
+  })();
+
+  // TODO: Always show InfoIcon once we have content for the popoverContent
+  const showInfoIcon = false;
+  const customInfoImg = (
+    <img
+      src={toStaticUrl("img/gene_overview/info_purple.svg")}
+      alt="heatmap info tip"
+      className={styles.infoImage}
+    />
+  );
+
+  if (!isLoading && error) {
+    return <ErrorLoading />;
+  }
+
+  return (
+    <article
+      className={`${styles.HeatmapTile} card_wrapper stacked-boxplot-tile`}
+    >
+      <div className="card_border container_fluid">
+        <h2 className="no_margin cardtitle_text">
+          Compound Viability
+          {showInfoIcon && (
+            <InfoIcon
+              target={customInfoImg}
+              popoverContent={<p>{"This is a tooltip"}</p>}
+              popoverId={`struc-detail-popover`}
+              trigger={["hover", "focus"]}
+            />
+          )}
+        </h2>
+        <div className="card_padding">
+          <div className={styles.subHeader}>
+            {compoundName} sensitivity distributed per dose
+          </div>
+          {(isLoading || isLoadingDataset) && !error && <PlotSpinner />}
+          {!isLoading && !error && sortedHeatmapFormattedData && (
+            <div className={styles.heatmapWithTriangle}>
+              <DoseTriangleLabel />
+
+              <div className={styles.heatmapContainer}>
+                <PrototypeBrushableHeatmap
+                  data={{
+                    ...sortedHeatmapFormattedData,
+                    x: sortedHeatmapFormattedData.x,
+                    y: sortedHeatmapFormattedData.y,
+                    z: sortedHeatmapFormattedData.z,
+                  }}
+                  onLoad={() => {}}
+                  xAxisTitle={"Cell Lines"}
+                  yAxisTitle={`Dose`}
+                  legendTitle={"Viability"}
+                  interactiveVersion={false}
```

Comment:
I'm really glad you were able to reuse this component without too much difficulty. And I like that it seems like it'll be possible to reuse this in the future as well if we incorporate heatmaps into other parts of the portal 😁